class-tgm-plugin-activation.php

```diff
@@ -488,11 +495,14 @@ public function filter_plugin_action_links_update( $actions ) {
 		 *
 		 * Down right easy once you know how...
 		 *
+		 * Returns early if not the TGMPA page.
+		 *
 		 * @since 2.1.0
 		 *
 		 * @global string $tab Used as iframe div class names, helps with styling
 		 * @global string $body_id Used as the iframe body ID, helps with styling
-		 * @return null Returns early if not the TGMPA page.
+		 *
+		 * @return void
```

Comment:
WP does not use return void. That was clarified last year. If there's no `return`, there's no `@return`.